### ISSUE TITLE
Can you add a function to open multiple resources in the resource manager?

### ISSUE THREAD

**Issue by AndroidSix** [cocos/cocos-engine#16465]
### Use Case

When I add multiple materials, I need to drag the corresponding picture for each material, but there are too many that I need to slide back and forth to find pictures. I hope that if the resource manager can open more, I don't have to slide back and forth.

### Problem Description

There are not many available now and it cannot meet my needs.

### Alternatives Considered

You can make a switch to determine the number of turns on

**Comment by minggo** [comment]
It is a requirement for editor, right?

**Comment by AndroidSix** [comment]
> It is a requirement for editor, right?

Yes, it's very inconvenient when editing model texture maps, and multitasking will meet my needs.
